user_journey/views/cohort_analysis/clients_last_seen.view.lkml

```diff
@@ -7,7 +7,15 @@ view: clients_last_seen {
         `moz-fx-data-shared-prod`.search.search_clients_last_seen AS clients_last_seen
       WHERE
         {% condition cohort_analysis.date %} CAST(DATE_SUB(clients_last_seen.submission_date, INTERVAL 56 DAY) AS TIMESTAMP) {% endcondition %}
-        OR clients_last_seen.submission_date = DATE_SUB(current_date, INTERVAL 2 DAY);;
+        OR clients_last_seen.submission_date = DATE_SUB(current_date, INTERVAL 2 DAY)
+        AND {% if days_since_message.days_since_message._is_selected %}
+          TRUE
+        {% else %}
+          ERROR("""
+            Must use the days_since_mesage.days_since_message dimension in this explore.
+            This explore CROSS JOINS with all possible days. Unless we GROUP BY each of those days,
+            we'll get incorrect results for the outputted measures.""")
+        {% endif %};;
```

Comment:
This lets us require that users select a field in the explore. It would be nice for Looker to support this natively but this will have to do in the meantime. If it's not selected, a big warning will show up in the explore with the error message.